src/lib/swapper/swappers/ThorchainSwapper/constants.ts

```diff
@@ -41,3 +41,7 @@ export const THORCHAIN_LONGTAIL_STREAMING_SWAP_SOURCE: SwapSource = `${SwapperNa
 // static automatic outbound fee as defined by: https://daemon.thorchain.shapeshift.com/lcd/thorchain/constants
 // expressed in thor units (8 decimals of precision)
 export const THORCHAIN_OUTBOUND_FEE_RUNE_THOR_UNIT = '2000000'
+
+export const QUOTER_CONTRACT_ADDRESS = '0xb27308f9F90D607463bb33eA1BeBb41C27CE5AB6' // FIXME: this is only true for Ethereum
```

Comment:
There's nothing inherently THORChain about this - it's the actual quoter contract of UNI-V3
```suggestion
export const UNI_V3_QUOTER_CONTRACT_ADDRESS = '0xb27308f9F90D607463bb33eA1BeBb41C27CE5AB6' // FIXME: this is only true for Ethereum
```

And probably remove the comment while at it too in profit of naming clarity e.g `UNI_V3_ETHEREUM_QUOTER_ADDRESS`

Reply:
It is only consumed by our THORChain swapper, was more the thought - only taking the constants up as high as they need to go.

Good call on incorporating the chain name into the var name itself 👌

Reply:
Also changed `POOL_FACTORY_CONTRACT_ADDRESS` to `UNI_V3_ETHEREUM_POOL_FACTORY_CONTRACT_ADDRESS` and kept at THORChain swapper root level as @0xApotheosis suggested I like the idea to put constants up as high as they need only